pkg/gatherers/clusterconfig/recent_metrics.go

```diff
@@ -45,6 +47,8 @@ const (
 //   - "virt_platform metric": 4.6.34+, 4.7.16+, 4.8+
 //   - "vsphere_node_hw_version_total": 4.7.11+, 4.8+
 //   - "console_helm_installs_total": 4.11+
+//   - "console_helm_upgrades_total": 4.11+
+//   - "console_helm_uninstalls_total": 4.11+
```

Comment:
Just a nit: We usually don't add this. We wait for the corresponding z-stream release and then we only update the versions in the master branch.